dashboard/db/migrate/20160617163547_create_user_geos.rb

```diff
@@ -0,0 +1,16 @@
+class CreateUserGeos < ActiveRecord::Migration
+  def change
+    create_table :user_geos do |t|
+      t.references :user, index: true, unique: true, foreign_key: true, null: false
+      t.integer :user_id
+      t.timestamps null: false
+      t.datetime :indexed_at, null: false
```

Comment:
It may not be, though it seems useful to have a non-rails managed timestamp that clearly indicates when the IP address was last indexed.

An example where it could be different is if we update our geocoder (on my plate). We'd reindex, though if there was no change, we wouldn't update.